development.md

```diff
@@ -10,8 +11,24 @@ Firefox version 38 or later can be found [here](https://nightly.mozilla.org/).
 
 For jpm installation, follow the instructions as mentioned [here](https://developer.mozilla.org/en-US/Add-ons/SDK/Tools/jpm#Installation).
 
+**Running**
+
 Once you have Firefox version 38 or later and jpm installed, run this command in the privly-jetpack directory -
 
 > jpm run -b /path/to/firefox/executable
- 
+
+Linux:
+
+> jpm run -b `which linux`
```

Comment:
@smcgregor If that's what it is, I'll change it and send a PR to privly/privly-jetpack ?